app/views/carts/checkout.html.haml

```diff
@@ -1,11 +1,16 @@
-Hey, you're checking out.
+- @headline = "Hey #{current_user.name}, you're checking out"
```

Comment:
I moved most to controller (couldnt with the devise pages?), and will research how to do what I want with a helper method.